File: event/src/test/java/de/danoeh/antennapod/event/PlayerErrorEventTest.java
package de.danoeh.antennapod.event;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNull;

public class PlayerErrorEventTest {

    @Test
    public void messageIsStored() {
        PlayerErrorEvent event = new PlayerErrorEvent("boom");
        assertEquals("boom", event.getMessage());
    }

    @Test
    public void nullMessageIsStoredWithoutThrowing() {
        PlayerErrorEvent event = new PlayerErrorEvent(null);
        assertNull(event.getMessage());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/PlayerErrorEventTest.kt
package de.danoeh.antennapod.event

import org.junit.Assert.assertEquals
import org.junit.Assert.assertNull
import org.junit.Test

class PlayerErrorEventTest {

    @Test
    fun messageIsStored() {
        val event = PlayerErrorEvent("boom")
        assertEquals("boom", event.message)
    }

    @Test
    fun nullMessageIsStoredWithoutThrowing() {
        val event = PlayerErrorEvent(null)
        assertNull(event.message)
    }
}
